src/Raven.Server/Documents/Indexes/Static/StaticIndexBase.cs

```diff
@@ -379,7 +379,7 @@ public IEnumerable<dynamic> Recurse(object item, Func<dynamic, dynamic> func)
             return new RecursiveFunction(item, func).Execute();
         }
 
-        protected IEnumerable<AbstractField> CreateField(string name, object value, CreateFieldOptions options)
+        protected dynamic CreateField(string name, object value, CreateFieldOptions options)
```

Comment:
Why are you doing this?

Reply:
Oh, fixed now. My mistake. I thought it was for spatial.